src/utils/getMethodDocumentation.js

```diff
@@ -42,10 +46,14 @@ function getMethodParamsDoc(methodPath, jsDoc) {
     const typePath = getTypeAnnotation(paramPath);
     if (typePath) {
       type = getFlowType(typePath);
+      if (types.GenericTypeAnnotation.check(typePath.node)) {
+        type.name = typePath.node.id.name;
+      }
```

Comment:
This check might be better in getFlowType. I can check, I couldn't remember why it is currently not working